src/app/features/course/course.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="course">
+  <div class="content">
+    <h3 class="title">{{ course.title }}</h3>
+    <p class="description">{{ course.description }}</p>
+  </div>
+  <div class="detailed-content">
+    <div class="detailed-content__item">
+      <span class="caption">Authors: </span>{{ course.authors.join(', ') }}
```

Comment:
`course.authors.join(', ')` - for this case it is okay, but for the future, try to avoid any functions inside template.

Reply:
fixed